### ISSUE TITLE
How to set up preset files with yabridge?

### ISSUE THREAD

**Issue by cbacott** [robbert-vdh/yabridge#261]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

Forgive me if this has been answered already. I've done some searching and cannot find any mention of this. 

I am having issues with preset files not being detected or loaded in my plugins, but only in certain cases. For instance, Valhalla reverb plugins aren't loading any presets, and neither is Kiive Tape Face. A wine process starts briefly when I try to go to the preset menus on these plugins, but then immediately exits, and I cannot see any error. I am not sure how yabridge is handling these preset files, or where to even put them really when I extract them from the Wine prefix. Is there any information or documentation about how this is supposed to be handled with yabridge? 

I most likely set up something wrong, I just didn't have any docs listing how to do this and probably got a path wrong. 

Thanks! 

### Operating system

Debian Bookworm

### How did you install yabridge?

Direct from GH releases page

### yabridge version

5.0.5

### yabridgectl version

5.0.5

### Wine version

wine-8.10

### Plugin type

VST3

### Plugin architecture

64-bit

### Host

Reaper

### Desktop environment or WM

Cinnamon

**Comment by robbert-vdh** [comment]
Are you sure they are not loading presets? You're describing a context menu opening and then immediately closing, which is something different. You forgot to fill in part of the template. I'm assuming you're using GNOME? You probably have some extension enabled that messes with focus or window activation.

**Comment by cbacott** [comment]
Apologies, I'm using Cinnamon. It's in the "native" windows UI that comes with the plugin. In one case, the plugin would display the presets in the menu, however it lists no presets. Attaching screenshot for ValhallaRoom to explain.

![Screenshot from 2023-07-04 15-10-36](https://github.com/robbert-vdh/yabridge/assets/19333095/cbca60f7-ebd1-4a7d-8193-460943e292fd)

In the other case, with Kiive TapeFace, I just realized it does indeed load the presets, but the menu for "Factory Presets" that I click on to show them doesn't appear but for a split second, and I can see a wine process in my task bar appear and disappear every time I click that menu item. I could try to run Reaper in a terminal to see if I get any debug output, but I am doubting so as it's spawning a wine process to handle the menu click and I don't know how to directly run that process in a terminal to get the output on what it's crashing on. Aside from that, I just realized there were some arrows that scroll through the available presets and that does appear to be working, I just didn't know that's what those arrows do. These VST UI designs aren't always intuitive. 

As it stands, disregard the menu-wine problem in TF, that's not anything related to yabridge, more a wine failure. I can see that it does have the presets somehow, though I don't know exactly what I did to make them be available. I am more interested in knowing how or where to put the preset files from the wine bottle to make them available via yabridge, as no presets are showing up in the Valhalla plugins.
